src/match.cpp

```diff
@@ -220,14 +224,12 @@ std::vector<MatchResult> regexbench::match(Engine& engine,
                                   (pLogger ? pLogger.get() : nullptr)));
   }
 
-  uint32_t sec = 0;
-  bool realTime = true;
+  gettimeofday(&begin, NULL);
```

Comment:
Yes. I will modify using `begintime` in `MatchResult` struct instead of local variable `begin`.